src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -0,0 +1,165 @@
+      </summary>
+      <remarks>
+        This transform produces a matrix of token ngrams/skip-grams counts for a given corpus of text.
+        The n-grams are represented as count vectors, with vector slots corresponding to n-grams.
+        Embedding ngrams in a vector space allows their contents to be compared in an efficient manner. 
+        The slot values in the vector can be weighted by the following factors:
+        <list>
+          <item><description>term frequency - The number of occurrences of the slot in the text</description></item>
+          <item><description>
+              inverse document frequency - A ratio (the logarithm of inverse relative slot frequency)
+              that measures the information a slot provides by determining how common or rare it is across the entire text.
+            </description></item>
+            <item><description>term frequency-inverse document frequency - the product term frequency and the inverse document frequency.</description></item>
+        </list>
+        This transform is not typically used on its own, but it is one of the transforms composing the <see cref="Microsoft.ML.Transforms.TextFeaturizer">Text Featurizer</see> .
+      </remarks>
+      <seealso cref="Microsoft.ML.Transforms.WordTokenizer"/>
+      <seealso cref="Microsoft.ML.Transforms.TextToKey"/>
+      <seealso cref="Microsoft.ML.Transforms.TextFeaturizer"/>
+      <seealso cref="Microsoft.ML.Transforms.CharacterTokenizer"/>
+      <example>
+        <code>
+          pipeline.Add(new NGramTranslator(&quot;TextColumn&quot;){ Weighting=NgramTransformWeightingCriteria.TfIdf  } );
+      </code>
+      </example>
+    </member>
+
+    <member name="SentimentAnalyzer">
+      <summary>
+        Uses a pretrained sentiment model to score input strings.
+      </summary>
+      <remarks>
+        <para>The Sentiment transform returns the probability that the sentiment of a natural text is positive. </para>
+        <para>
+          The model was trained with the <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">Sentiment-specific word embedding (SSWE)</a>  and NGramFeaturizer on Twitter sentiment data,
+          similarly to the sentiment analysis part of the
+          <a href="https://www.microsoft.com/cognitive-services/en-us/text-analytics-api">Text Analytics cognitive service</a>. 
+          The transform outputs a score between 0 and 1 as a sentiment prediction 
+          (where 0 is a negative sentiment and 1 is a positive sentiment).</para> 
+          <para>Currently it supports only English.</para>
+      </remarks>
+      <example>
+        <code>
+          pipeline.Add(new SentimentAnalyzer(){ Source = &quot;TextColumn&quot; }  );
+        </code>
+      </example>
+    </member>
+
+    <member name="CharacterTokenizer">
+      <summary>
+        This transform breaks text into individual tokens, each consisting of individual character.
```

Comment:
>individual character [](start = 78, length = 20)

*an* individual character

Reply:
reopening since I don't see the change

---
In reply to: [202757869](https://github.com/dotnet/machinelearning/pull/529#discussion_r202757869) [](ancestors = 202757869)

Reply:
welllll I didn't add the an, as suggested, just added the s at the end of the word: "character" so the sentence is grammatically correct now. 

---
In reply to: [203203395](https://github.com/dotnet/machinelearning/pull/529#discussion_r203203395) [](ancestors = 203203395,202757869)

Reply:
I still think Tom's wording is clearer

---
In reply to: [203204359](https://github.com/dotnet/machinelearning/pull/529#discussion_r203204359) [](ancestors = 203204359,203203395,202757869)

Reply:
fixed, as Tom suggested, to properly convey the length of the characters in the tokens. 

---
In reply to: [203205489](https://github.com/dotnet/machinelearning/pull/529#discussion_r203205489) [](ancestors = 203205489,203204359,203203395,202757869)